### PR TITLE
Omar-ToDo (CRUD)

### PR DESCRIPTION
**Pull request by omrmhd5**
Implemented CRUD Operations Utilizing DB

Refactored The CRUD Codes into the Controller

Handled bugs and errors regarding spamming buttons and in case of no network

Styling CSS

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,28 +1,11 @@
...
-      updated_at: "",
-    },
-    {
-      id: 39,
-      title: "task 3",
-      completed: false,
-      created_at: "",
-      updated_at: "",
-    },
-  ],
```
Comment (DaliaAmr) [inline index.js:24]: todos: [],
Remove the above lines and only add this line.

**Review by DaliaAmr — APPROVED** [review]
Check the comments in the code, but overall everything is good.

**Review by zeyad547 — COMMENTED** [review]
Great work.

**Review by rawanramadanseddik — CHANGES_REQUESTED** [review]
Please remove all the "console.logs()" 's, they are useless

**Comment by omrmhd5** [comment]
> Please remove all the "console.logs()" 's, they are useless

No

**Review by youssefibrahim22 — APPROVED** [review]
all good

**Pull request merged**